### PR TITLE
Update changes

### PR DESCRIPTION
**Pull request by kboygit**
Update spacing and small changes

### CODE REVIEW

File: app/views/mentee_mailer/welcome.html.erb [hunk 1]
```diff
@@ -8,12 +8,12 @@ A little bit about myself:<br />
 </p>
 
 <p>
-The best way to start this process is for me to learn more about you. Getting an understanding of your military background and your programming goals,
+The best way to start this process is for me to learn more about you.Getting an understanding of your military background and your programming goals,
 whether they be pursuing a new career, furthering an existing career, or simply learning to code as a hobby, will be helpful in matching you up with the
-right mentoring groups. It’d be great for us to connect on an initial call so I can go over our “checklist” with you. Would you be available for a quick chat?
+right mentoring groups.It’d be great for us to connect on an initial call so I can go over our “checklist” with you. Would you be available for a quick chat?
```
Comment (rickr) [inline app/views/mentee_mailer/welcome.html.erb]: I think the space between `groups.` and `It'd` got removed here.

File: app/views/mentee_mailer/welcome.html.erb [hunk 2]
```diff
@@ -8,12 +8,12 @@ A little bit about myself:<br />
...
 </p>
 
 <p>
-Look forward to hearing from you!<br />
+Looking forward to hear from you!<br />
```
Comment (rickr) [inline app/views/mentee_mailer/welcome.html.erb]: This change doesn't make much sense grammatically. Maybe `Looking forward to hearing from you` instead?

Comment (kboygit) [inline app/views/mentee_mailer/welcome.html.erb]: I agree sir

**Comment by rickr** [comment]
Hello @kboygit - are you able to make the requested changes?

**Comment by kboygit** [comment]
I thought I already requested for this sorry, I just saw this on my email

**Comment by rickr** [comment]
Closing this due to inactivity. Please feel free to reopen when the requested changes have been made!

**Pull request closed without merging**